webcrypto/tests/import_export/symmetric.js

```diff
@@ -0,0 +1,257 @@
+// This file contains an adaptation of the import_export/symmetric_importKey.https.any.js
+// implementation from the W3C WebCrypto API test suite.
+//
+// Some of the function have been modified to support the k6 javascript runtime,
+// and to limit its dependency to the rest of the W3C WebCrypto API test suite internal
+// codebase.
+//
+// The original implementation is available at:
+// https://github.com/web-platform-tests/wpt/blob/8cf108d380ee91ef2667a05dd2389ad5cd6d6529/WebCryptoAPI/import_export/symmetric_importKey.https.any.js
+
+
+// Test importKey and exportKey for non-PKC algorithms. Only "happy paths" are
+// currently tested - those where the operation should succeed.
+
+var subtle = crypto.subtle;
+
+// keying material for algorithms that can use any bit string.
+var rawKeyData = [
+    new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]),
+    new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
+                    17, 18, 19, 20, 21, 22, 23, 24]),
+    new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,
+                    17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32])
+];
+
+// combinations of algorithms, usages, parameters, and formats to test
+var testVectors = [
+    {name: "AES-CTR",               legalUsages: ["encrypt", "decrypt"],      extractable: [true, false], formats: ["raw"]},
+    // {name: "AES-CBC",               legalUsages: ["encrypt", "decrypt"],      extractable: [true, false], formats: ["raw"]},
+    // {name: "AES-GCM",               legalUsages: ["encrypt", "decrypt"],      extractable: [true, false], formats: ["raw"]},
```

Comment:
Aren't the CBC and GCM AES modes [already supported](https://github.com/grafana/xk6-webcrypto/blob/b0438aece9723a829c93af408e103eb108a788a2/webcrypto/subtle_crypto.go#L475) in the raw format?

Tests pass when I uncomment these 2 lines.

Reply:
Completely forgot to uncomment those two lines 🤦🏻 , good catch 👀